404 error in new install

Thanks for creating this great package! I'm trying to migrate from xo and found your repo [here](https://github.com/xojs/xo/issues/702#issuecomment-1437219821).

I'm getting this error on `npx create-sheriff-config`:

```
Need to install the following packages:
create-sheriff-config@5.1.2
Ok to proceed? (y) y
npm ERR! code E404
npm ERR! 404 Not Found - GET https://registry.npmjs.org/@sherifforg%2fconstants - Not found
npm ERR! 404
npm ERR! 404  '@sherifforg/constants@0.0.1' is not in this registry.
npm ERR! 404
npm ERR! 404 Note that you can also install from a
npm ERR! 404 tarball, folder, http url, or git url.
```

--- a/.changeset/neat-owls-buy.md
+++ b/.changeset/neat-owls-buy.md
@@ -1,0 +1,6 @@
+---
+"create-sheriff-config": minor
+"eslint-config-sheriff": minor
+---
+
+fix(deps): moved @sherifforg/constants to devDeps. Fixes #150

--- a/packages/create-sheriff-config/package.json
+++ b/packages/create-sheriff-config/package.json
@@ -15,7 +15,6 @@
   },
   "license": "MIT",
   "dependencies": {
-    "@sherifforg/constants": "workspace:*",
     "cli-highlight": "^2.1.11",
     "consola": "^3.2.3",
     "find-up": "^6.3.0",
@@ -30,6 +29,7 @@
     "create-sheriff-config": "./dist/index.js"
   },
   "devDependencies": {
+    "@sherifforg/constants": "workspace:*",
     "@types/lodash-es": "^4.17.12",
     "@types/node": "^20.10.6",
     "@types/yargs": "^17.0.32",

--- a/packages/create-sheriff-config/src/utils/createFile.ts
+++ b/packages/create-sheriff-config/src/utils/createFile.ts
@@ -1,6 +1,6 @@
+import { writeFile } from 'node:fs';
 import { consola } from 'consola';
 import { colors } from 'consola/utils';
-import { writeFile } from 'fs';
 import { throwError } from './throwError';
 
 export const createFile = (

--- a/packages/eslint-config-sheriff/package.json
+++ b/packages/eslint-config-sheriff/package.json
@@ -60,7 +60,6 @@
     "@eslint/js": "^8.57.0",
     "@next/eslint-plugin-next": "^13.2.3",
     "@regru/eslint-plugin-prefer-early-return": "^1.0.0",
-    "@sherifforg/constants": "workspace:*",
     "@stylistic/eslint-plugin": "^1.5.1",
     "astro-eslint-parser": "^0.16.3",
     "eslint-config-flat-gitignore": "^0.1.3",
@@ -97,6 +96,7 @@
   "license": "MIT",
   "devDependencies": {
     "@arethetypeswrong/cli": "^0.9.0",
+    "@sherifforg/constants": "workspace:*",
     "@sherifforg/types": "workspace:*",
     "@types/eslint-config-prettier": "^6.11.3",
     "@types/lodash": "^4.14.191",

--- a/turbo.json
+++ b/turbo.json
@@ -1,5 +1,5 @@
 {
-  "$schema": "https://turbo.build/schema.json",
+  "$schema": "https://turbo.build/schema.v1.json",
   "globalDependencies": ["**/.env.*local"],
   "pipeline": {
     "topo": {

